CONTRIBUTING.md

```diff
@@ -189,6 +189,9 @@ We use 3 blocks of imports in our Rust files:
 
 We believe that this makes it easier to see where a particular import comes from.
 
+We prefer to import core traits, e.g. using `TryFrom` instead of
```

Comment:
`TryFrom` is a bad example, because it has been a part of the prelude since the 2021 edition (that is, it doesn't need to be imported at all but can still be referenced directly without qualifying).

I think this isn't really particular to `core` and more broadly holds: I think our style is that we generally prefer referencing types and traits by name directly without qualifying because qualifying adds some noise that is generally unnecessary. Except if the symbol is overly generic/easily confused between different crates (for example, `Error` will generally refer to the local crate's `Error` type, so we alias `std::error::Error` to `StdError` and refer to `io::Error` using a one-level qualified path -- benefitting from the short module name to avoid too much verbosity).

Reply:
Reworked to not be specific to `core`, borrowing your examples.